config/locales/es.yml

```diff
@@ -20,6 +20,7 @@ es:
     record_not_found: '%{model} no encontrado'
     missing_param: 'Falta el siguiente parámetro o el valor está vacío: %{param}'
     invalid_content_type: 'Content-Type inválido en el encabezado'
+    reset_password_token_invalid: 'el token para el reseteo de contraseña no es valido'
```

Comment:
Can you move this translation below `page_not_found` to have the same order as the `en.yml` file?